CMakeLists.txt

```diff
@@ -233,7 +233,9 @@ else()
   endif()
 endif()
 
-check_linker_flag(CXX "-Wl,--no-undefined" SNMALLOC_LINKER_SUPPORT_NO_ALLOW_SHLIB_UNDEF)
+if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD")
+  check_linker_flag(CXX "-Wl,--no-undefined" SNMALLOC_LINKER_SUPPORT_NO_ALLOW_SHLIB_UNDEF)
```

Comment:
Shouldn't this just failed to find the flag on OpenBSD if it doesn't support the flag?

Reply:
in fact it s supported but disabled on purpose here, it does not have the same behavior than on Linux.